File: examples/dotNET/2. Functions/Binary Analysis/Basic Particle/vb/ExampleImagesFolder.vb
Imports System
Imports Microsoft.Win32

''' <summary>
''' Class to get the NI Vision example images folder
''' </summary>
Public NotInheritable Class ExampleImagesFolder
	Private Sub New()
	End Sub
	Public Shared Function GetExampleImagesFolder() As String
	    Dim key As RegistryKey = Registry.LocalMachine
	    key = key.OpenSubKey("SOFTWARE\National Instruments\IMAQ Vision\CurrentVersion")
        Dim imagesPath As String = "C:\\Users\\Public\\Documents\\National Instruments\\Vision\\Examples\\Images"
        key.Close()
	    Return imagesPath
	End Function
End Class

File: examples/dotNET/2. Functions/Binary Analysis/Basic Particle/cs/ExampleImagesFolder.cs
using System;
using Microsoft.Win32;

namespace NationalInstruments.Vision
{
    /// <summary>
    /// Class to get the NI Vision example images folder
    /// </summary>
    public static class ExampleImagesFolder
    {
        public static string GetExampleImagesFolder()
        {
            RegistryKey key = Registry.LocalMachine;
            key = key.OpenSubKey(@"SOFTWARE\National Instruments\IMAQ Vision\CurrentVersion");
            string imagesPath = "C:\\Users\\lvadmin\\Desktop\\New folder\\Images";
            key.Close();
            return imagesPath;
        }
    }
}
